<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户登录</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>360</y>
     <width>400</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;
background-color: rgb(196, 196, 196);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>215</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;新宋体&quot;;</string>
   </property>
   <property name="text">
    <string>手机号码</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;新宋体&quot;;</string>
   </property>
   <property name="text">
    <string>验证码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>215</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Adobe Arabic&quot;;
color: rgb(2, 2, 2);</string>
   </property>
   <property name="text">
    <string>手机验证</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Adobe Arabic&quot;;
color: rgb(90, 90, 90);</string>
   </property>
   <property name="text">
    <string>请输入您的手机号码，</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Adobe Arabic&quot;;
color: rgb(90, 90, 90);</string>
   </property>
   <property name="text">
    <string>登录或注册您的摩拜单车账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reqVerifyCodeButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>590</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;新宋体&quot;;</string>
   </property>
   <property name="text">
    <string>----------------- 使用社交账号一键登录 -------------------------</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>541</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="wechatButton">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="weiboButton">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>loginButton</zorder>
  <zorder>accountLabel</zorder>
  <zorder>passwordLabel</zorder>
  <zorder>accountEdit</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>reqVerifyCodeButton</zorder>
  <zorder>label_4</zorder>
  <zorder>horizontalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
